### ISSUE TITLE
tools.graph_transforms.TransformGraph has no docs or example of usage

### ISSUE THREAD

**Issue by Uiuran** [tensorflow/tensorflow#31415]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:

the provided wrapper function has no docs

## Description of issue (what needs changing):

provide a docs such
examples of usage:
TransformGraph( graph.as_graph_def(), [], [], ['remove_nodes(op=loss/init)']) ...

### Clear description

I wanna to use this method has a way to do specifics editions and graph redefinitions while building the model, from inside python, withou having to go to command line.

### Parameters defined

I think how to use the parameters are exactly the problem, the README.md from the repository gives bazel example, but it dosnt work as it should in the wrapper
### Usage example

this is my first try, i could not get the desirable result (strip init op from graph):

```python
graph = tf.Graph()
with graph.as_default():

  with tf.variable_scope('signal_in'):
    signal_in = tf.placeholder(tf.float32, shape=(10,40,2,1))

  with tf.variable_scope('dascope1'):
    conv_linear = tf.keras.layers.Conv2D( 8, (8,2), padding='valid', name='conv_linear', use_bias=True, kernel_initializer=tf.initializers.lecun_normal(seed=137), bias_initializer=tf.initializers.lecun_normal(seed=137) )(signal_in)
  
  with tf.variable_scope('softmax'):
    logits = tf.contrib.layers.fully_connected(conv_linear, 2, activation_fn=None, normalizer_fn=None, normalizer_params=None, weights_initializer=tf.initializers.lecun_normal(seed=731), weights_regularizer=None, biases_initializer=tf.initializers.lecun_normal(seed=777), biases_regularizer=None, reuse=None, variables_collections=None, outputs_collections=None, trainable=True, scope='logit')
    softmax = tf.nn.softmax(logits,axis=0)            
    
  with tf.variable_scope('loss'):
    l_vec = tf.placeholder(tf.float32, shape=(10,2))
    loss = tf.keras.losses.CategoricalCrossentropy(from_logits=False, label_smoothing=0)(l_vec, softmax)         
    minimize_op = tf.train.AdamOptimizer(learning_rate=0.05).minimize(loss)
    tf.global_variables_initializer()
```
then:

```python
graphdef = tf.tools.graph_transforms.TransformGraph( graph.as_graph_def(), [], [], ['remove_nodes(op=loss/init)'])

with tf.Graph().as_default() as g:  
  tf.import_graph_def(graphdef,name = '')
  for op in g.get_operations():
    if op.name.split('/')[-1] == 'init':
      print('True')
``` 

returns True
So how to use this wrapper ? note that init op dosnt have any input output but only dependency arrows as input.

### Request visuals, if applicable

Are there currently visuals? If not, will it clarify the content?

### Submit a pull request?

Waiting for instructions of the community about the use of this function.

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by jvishnuvardhan** [comment]
Please fill the issue [template](https://github.com/keras-team/keras/issues/new?template=a--tensorflow-backend-users.md). Could you update them if they are relevant in your case, or leave them as N/A? Along with the template, please provide a standalone code to reproduce the issue. Thanks!

**Comment by Saduf2019** [comment]
@Uiuran Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by MarkDaoust** [comment]
There's an extensive "readme" in the source directory: 

https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/graph_transforms

But we can't add this to tensorflow.org as it is not considered an implementation detail and not part of the public API.
